x/circuit/README.md

```diff
@@ -101,7 +101,7 @@ Reset is called by an authorized account to enable execution for a specific msgU
 ### MsgAuthorizeCircuitBreaker
 
 ```protobuf reference
-https://github.com/cosmos/cosmos-sdk/blob/main/proto/cosmos/circuit/v1/tx.proto#L25-L75
+https://github.com/cosmos/cosmos-sdk/blob/381de6452693a9338371223c232fba0c42773a4b/proto/cosmos/circuit/v1/tx.proto#L26-L40
```

Comment:
Can we link the last commit from main? Those protos are under the module path now, so it would be god to reflect that.

Reply:
yes, thank you for feedback, i updated